config/database.php

```diff
@@ -96,6 +96,7 @@
                 PDO::MYSQL_ATTR_SSL_CERT                => env('DB_SSL_CERT_PATH'), // /path/to/cert.pem
                 PDO::MYSQL_ATTR_SSL_CA                  => env('DB_SSL_CA_PATH'),   // /path/to/ca.pem
                 PDO::MYSQL_ATTR_SSL_CIPHER              => env('DB_SSL_CIPHER'),
+                PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT  => env('DB_SSL_VERIFY_SERVER'), //true/false
```

Comment:
Should we perhaps default this to something sane if folks don't add this value to their .env?

Reply:
Good point, I wasn't thinking about upgrading existing installs without updating the .env file. Let me test the behavior when the value is set to `null`.

Reply:
Also check if the variable doesn't even exist in the .env :)